test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/staking/StakingSuite.java

```diff
@@ -74,11 +74,12 @@ public boolean canRunConcurrent() {
     public List<HapiApiSpec> getSpecsInSuite() {
         return List.of(
                 new HapiApiSpec[] {
+                    canOverrideMaxMinStakeRatios(),
                     // Need to run each separately
-                    rewardsWorkAsExpected(),
-                    rewardPaymentsNotRepeatedInSamePeriod(),
-                    getInfoQueriesReturnsPendingRewards(),
-                    secondOrderRewardSituationsWork()
+                    //                    rewardsWorkAsExpected(),
+                    //                    rewardPaymentsNotRepeatedInSamePeriod(),
+                    //                    getInfoQueriesReturnsPendingRewards(),
+                    //                    secondOrderRewardSituationsWork()
```

Comment:
can be uncommented

Reply:
Cleaned this file up and included a more meaningful `canBeRewardedWithoutMinStakeIfSoConfigured()` EET.